ci/tasks/build-bosh-release.yml

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    # TODO mattysweeps: https://github.com/cloudfoundry-community/stackdriver-tools/issues/223
+    repository: m0pt0pmatt/tile-generator
+inputs:
+  - name: stackdriver-tools-source
+outputs:
+  - name: bosh-release-out
+run:
+  dir: stackdriver-tools-source
+  path: sh
+  args: [-c, "apk add --no-cache make git && pwd && make bosh-release && cp build/stackdriver-tools-release-*.tar.gz* ../bosh-release-out/"]
```

Comment:
Suggest considering splitting out the shell script into its own file as a convention.